src/interfaces/mining.h

```diff
@@ -89,15 +89,16 @@ class Mining
 
     /**
      * Waits for the connected tip to change. During node initialization, this will
-     * wait until the tip is connected.
+     * wait until the tip is connected (regardless of `timeout`).
```

Comment:
In commit "Have createNewBlock() wait for a tip" (db14ca3556ca792546bf4343feb733271333690f)

Commit message is a little confusing because it doesn't mention the timeout change. Would be clearer if it said the commit was changing two things (1) returning null on shutdown instead of last tip (2) ignoring timeout value during startup instead of returning 0 if timeout elapsed before tip was connected

Reply:
I improved the commit message.

Reply:
Will do if I need to retouch.